kernel/typeops.ml

```diff
@@ -590,3 +661,20 @@ let judge_of_case env ci pj cj lfj =
   let lf, lft = dest_judgev lfj in
   make_judge (mkCase (ci, (*nf_betaiota*) pj.uj_val, cj.uj_val, lft))
              (type_of_case env ci pj.uj_val pj.uj_type cj.uj_val cj.uj_type lf lft)
+
+(* Building type of primitive operators and type *)
+
+open CPrimitives
+
+let check_primitive_type env op_t t =
+  match op_t with
+  | OT_type PT_int63 ->
+    (try
+      default_conv ~l2r:false CUMUL env t mkSet
```

Comment:
Isn't this the wrong order? We want `Set <= t` right?
Indeed
~~~coq
Primitive int : Type := #int63_type.
(* Error: Was expecting the sort of this primitive type to be Set *)
~~~
Same for the other default_conv call below.

Reply:
Good catch, I got confused. Now fixed!